settings.gradle

```diff
@@ -59,3 +59,13 @@ subprojects.files.each { File f ->
         project(projectName).projectDir = projectFolder
     }
 }
+
+FileTree processProjects = fileTree("$rootDir/processing").include('**/build.gradle')
```

Comment:
It may be best to use

`subprojects = filetree(...)`

as the processProjects is not currently visible in build.gradle, unless you add logic in build.gradle to pull in the sensorML descriptions and add them to the deployment configuration like subprojects